contrib/cli/great_expectations_contrib/commands.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+import sys
+from collections import namedtuple
+
+import click
+from cookiecutter.main import cookiecutter
+
+Command = namedtuple("Command", ["name", "args", "error_message"])
```

Comment:
Very minor, but maybe consider renaming `args` to `full_command` since the 2nd item of each Command is the name and the args combined. Or Remove the name from 2nd item (i.e. instead of `black --check .` use `--check .`) and update `run_command` to prepend the name to the list passed to `subprocess.run(`

Reply:
That's a great point! I only added `name` because there's one case with `wheel` where the first command is `python`. 

We'll want to add more checks and they won't always correspond with the first term. I'll rename for now and we can re-evaluate as we add more items.